.github/pull_request_template.md

```diff
@@ -0,0 +1,37 @@
+[Jira ticket](https://dsdmoj.atlassian.net/browse/EL-XXX)
+
```

Comment:
`*Trello (if applicable): [Insert Trello link]* ` Not all OGD use Jira, some may use Trello

Reply:
have amended